admin/tests/acceptance/authenticated/users/list_test.js

```diff
@@ -0,0 +1,77 @@
+import { module, test } from 'qunit';
+import { currentURL, fillIn } from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import { authenticateAdminMemberWithRole } from 'pix-admin/tests/helpers/test-init';
+import { clickByName, visit } from '@1024pix/ember-testing-library';
+
+module('Acceptance | authenticated/users | list', function (hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+
+  module('when clicking on "Vider" button', function () {
+    test('should empty url params', async function (assert) {
+      // given
+      await authenticateAdminMemberWithRole({ isSuperAdmin: true })(server);
+
+      // when
+      await visit('/users/list?firstName=emma&lastName=sardine&email=emma@example.net');
+      await clickByName('Vider les champs de recherche');
+
+      // then
+      assert.strictEqual(currentURL(), '/users/list');
+    });
+
+    test('should empty all search fields', async function (assert) {
```

Comment:
Sur le modèle des filtres sur orga, on a laissé juste un test d'acceptance qui vérifie le changement d'url `orga/tests/acceptance/campaign-list-my-campaigns_test.js`
- un TU sur l'action du controller `orga/tests/unit/controllers/authenticated/campaigns/list/my-campaigns_test.js`
- des tests d'intégration `orga/tests/integration/components/campaign/filter/filters_test.js`
pour éviter d'avoir un test d'acceptance.

Qu'est ce qu'on peut faire sur admin ?

Reply:
Très bonne question ! J'aurai envie de faire un test unitaire sur la méthode `clearSearchFields` pour éviter d'écrire autant de tests d'acceptance .... mais le problème c'est que ça ne couvrirait plus les changements que j'ai du faire dans l'hbs (ex: `value={{this.firstNameForm}}`). 
Je peux quand même remplacer les 2 derniers tests en unitaire, mais le comportement sera cassé si on enlève `value={{this.lastNameForm}}` et on ne le verrait pas dans les tests ... 🤔

Reply:
vu ensemble , on laisse en l'état